src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier supplier = new ColorSupplier();
+    private final Random random = new Random();
+    private final int maxValue = 100;

```

Comment:
Follow naming convention of constants.
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
    private static final int MAX_VALUE = 100;    
```